linode/manager
# refactor: M3-6185: Standardize Query String Parsing and Stringifying

## Description 📝

- Currently we use `querystring` (from Node.js) and package called `qs` to handle query string manipulation and parsing. This is bad because `querystring` is from Node.js and we don't want to have to deal with polyfills and we don't want to use `qs` because we don't want extra dependences. 
- We really need to be using `URLSearchParams` because it is built into the browser. 
- We need to make this change because Vite will complain about us using a Node.js package in code intended for the browser
- Switch instances of `querystring` and `qs` to use `URLSearchParams`
- Remove any `querystring` or `qs` related packages
 
![Screenshot 2023-02-21 at 2 54 53 PM](https://user-images.githubusercontent.com/115251059/220463796-d3691fbf-3483-4749-986c-081ca9073d4e.jpg)

## How to test 🧪

```sh
yarn test queryParams
```

- When you scroll through the diff of this PR, check the corresponding functionality in Cloud Manager
